pages/clustering/replication/system-replication.mdx

```diff
@@ -1,28 +1,18 @@
 ---
-title: Experimental replication
+title: System replication
```

Comment:
This whole page should be deleted. NOTE: Try to move the information from this page under `Replication` page (if applicable, just to mention that databases+users+roles are also replicated.